openquakeplatform/openquakeplatform/ript/static/ript/js/site_conds.js

```diff
@@ -0,0 +1,283 @@
+        } else if (argument != 'none' && valueArg !== undefined) {
+            if (checkForValueInHeader(sc_obj.header, valueArg) == -1) {
+                sc_obj.header.push(valueArg);
+            }
+        }
+    }
+
+    // Get info from the expsure form and use it to build the table header
+    $('.sc_gid #costStruc option:selected').each(function() {
+        checkForValue($(this).attr('value'), 'structural');
+    });
+
+    $('.sc_gid #costNonStruc option:selected').each(function() {
+        checkForValue($(this).attr('value'), 'non-structural');
+    });
+
+    $('.sc_gid #costContent option:selected').each(function() {
+        checkForValue($(this).attr('value'), 'contents');
+    });
+
+    $('.sc_gid #costBusiness option:selected').each(function() {
+        checkForValue($(this).attr('value'), 'business');
+    });
+
+    $('.sc_gid #limitSelect option:selected').each(function() {
+        checkForValue($(this).attr('value'), 'limit');
+    });
+
+    $('.sc_gid #deductibleSelect option:selected').each(function() {
+        checkForValue($(this).attr('value'), 'deductible');
+    });
+
+    var perAreaVisible = $('.sc_gid #perArea:visible').length;
+    if (perAreaVisible === 1) {
+        sc_obj.header.push('area');
+    }
+
+    $('.sc_gid #occupantsCheckBoxes input:checked').each(function() {
+        sc_obj.header.push($(this).attr('value'));
+        // unfocus the selection menu, see the note at the exposure costStruc change event
+        $(this).blur();
+    });
+
+    $('.sc_gid #retrofittingSelect input:checked').each(function() {
+        sc_obj.header.push($(this).attr('value'));
+        // unfocus the selection menu, see the note at the exposure costStruc change event
+        $(this).blur();
+    });
+
+    var headerLength = sc_obj.header.length;
+
+    // Create the table
+    var container = document.getElementById('table');
+
+    ///////////////////////////////
+    /// Exposure Table Settings ///
+    ///////////////////////////////
+    $('.sc_gid #table').handsontable({
+        colHeaders: sc_obj.header,
+        rowHeaders: true,
+        contextMenu: true,
+        startRows: 3,
+        startCols: headerLength,
+        maxCols: headerLength,
+        className: "htRight"
+    });
+    sc_obj.tbl = $('.sc_gid #table').handsontable('getInstance');
+
+    $('.sc_gid #outputText').empty();
+    $('.sc_gid #convertBtn').show();
+}
+
+$('.sc_gid #downloadBtn').click(function() {
+    sendbackNRML(sc_obj.nrml, 'sc');
+});
+
+$('.sc_gid #convertBtn').click(function() {
+    // Get the values from the table
+    var data = sc_obj.tbl.getData();
+
+    var sites = '';
+    // Check for null values
+    for (var i = 0; i < data.length; i++) {
+        sites += '\t<site lon="' + data[i][0] + '" lat="' + data[i][1] + '" vs30="' + data[i][2] +
+             '" vs30Type="' + data[i][3] + '" z1pt0="' + data[i][4] + '" z2pt5="' + data[i][5] +'"/>\n';
+    }
+
+// <?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please make sure if you want to keep it as an example (in case, perhaps add a comment) or if we can remove it